aicsimageio/readers/bfio_reader.py

```diff
@@ -0,0 +1,258 @@
+
+                raise IndexError(
+                    "Scene id: Cannot change scene for "
+                    + f"{self.__class__.__name__} objects."
+                )
+
+        else:
+            raise TypeError(
+                f"Must provide either a string (for scene id) "
+                f"or integer (for scene index). Provided: {scene_id} ({type(scene_id)}."
+            )
+
+    @property
+    def channel_names(self) -> Optional[List[str]]:
+
+        return self._rdr.channel_names
+
+    @property
+    def physical_pixel_sizes(self) -> PhysicalPixelSizes:
+        return types.PhysicalPixelSizes(
+            self._rdr.ps_z[0],
+            self._rdr.ps_y[0],
+            self._rdr.ps_x[0],
+        )
+
+    def _read_delayed(self) -> xr.DataArray:
+
+        return self._general_data_array_constructor(
+            da.from_array(self._rdr, chunks=(1024, 1024) + (1,) * (self._rdr.ndim - 2)),
+            self._tiff_tags(),
+        )
+
+    def _tiff_tags(self) -> Optional[Dict[str, str]]:
+
+        tiff_tags: Optional[Dict[str, str]] = None
+        if self.backend == "python":
+            # Create a copy since TiffTags are not serializable
+            tiff_tags = {
+                code: tag.value
+                for code, tag in self._rdr._backend._rdr.pages[0].tags.items()
+            }
+
+        return tiff_tags
+
+    def _read_immediate(self) -> xr.DataArray:
+        return self._general_data_array_constructor(
+            self._rdr[:],
+            self._tiff_tags(),
+        )
+
+    @property
+    def ome_metadata(self) -> OME:
+        return self._rdr.metadata
+
+
+class OmeTiledTiffReader(BfioReader):
+    """
+    Wrapper around bfio.BioReader(backend="python").
+
+    Parameters
+    ----------
+    image: types.PathLike
+        Path to image file.
+    chunk_dims: List[str]
+        Which dimensions to create chunks for.
+        Default: DEFAULT_CHUNK_DIMS
+        Note: Dimensions.SpatialY, Dimensions.SpatialX, and DimensionNames.Samples,
+        will always be added to the list if not present during dask array
+        construction.
+    clean_metadata: bool
+        Should the OME XML metadata found in the file be cleaned for known
+        AICSImageIO 3.x and earlier created errors.
+        Default: True (Clean the metadata for known errors)
+
+    Notes
+    -----
+    If the OME metadata in your file isn't OME schema compliant or does not validate
+    this will fail to read your file and raise an exception.
+
+    If the OME metadata in your file doesn't use the latest OME schema (2016-06),
+    this reader will make a request to the referenced remote OME schema to validate.
+    """
+
+    backend: str = "python"
+
+    @staticmethod
+    def _is_supported_image(fs: AbstractFileSystem, path: str, **kwargs: Any) -> bool:
+        try:
+            # with fs.open(path) as open_resource:
+            with BioReader(path, backend="python"):
+
+                return True
```

Comment:
I am tempted to ask for two things:

1. Because this reader doesn't work for multi-scenes it may be good to fast fail in this function if the file is multi-scene.
2. Can you follow the non local filesystem error / warning pattern we have: https://github.com/AllenCellModeling/aicsimageio/blob/main/aicsimageio/readers/czi_reader.py#L83